src/lib/car.js

```diff
@@ -59,6 +59,35 @@ async function publishCar({ car, logger, queue = config.notificationsQueue }) {
   await publish({ queue, message: car.id, logger })
 }
 
+/**
+ * Broadcast IndexerComplete event on events pubsub topic
+ * @param {object} options
+ * @param {object} options.car
+ * @param {URL} options.car.url - url of CAR
+ * @param {URL} options.car.contentLength - byte length of CAR file
+ * @param {object} options.indexing - describes the indexing process
+ * @param {Date} options.indexing.startTime - when indexing began
+ * @param {Date} options.indexing.endTime - when indexing completed
+ */
+async function notifyIndexerCompletedEvent({
```

Comment:
Looks like this overlaps with `publishCar`

Reply:
@simone-sanfratello I wondered that, but since there were no comments on that function, I wasn't sure what what its intended scope was.

Reply:
We don't have a reason for keeping the "notify" queue now that we have the "events" topic. They have the same purpose.